client/src/helpers/unique-id.js

```diff
@@ -0,0 +1,6 @@
+function uniqueID() {
```

Comment:
also rename the file, and its calls
```suggestion
function createUniqueID() {
```

Reply:
done